src/tacticals.sig

```diff
@@ -1,17 +1,84 @@
+(* A collection of common, system agnostic tactics. This is the
+ * payoff for types implementing LCF. These tactics can be used
+ * to compositionally build up sophisticated logic programs from
+ * collection of small, system-dependent tactics.
+ *
+ * In order to actually use these, take the LCF ascribing module
+ * describing your system and feed it into the Tacticals functor.
```

Comment:
The signature should be agnostic about its implementation; let's remove the mention of the `Tacticals` functor.